stripper/ze_isla_nublar_p3.cfg

```diff
@@ -10,4 +10,49 @@ modify:
 	{
 		"solid" "0"
 	}
+}
+
+;Less of a bore-fest. 320 -> 190 -> 160
+modify:
+{
+	match:
+	{
+		"classname" "math_counter"
+		"targetname" "trexhitbased1"
+	}
+	replace:
+	{
+		"startvalue" "1000"
+	}
+}
+
+modify:
+{
+	match:
+	{
+		"classname" "trigger_multiple"
+		"targetname" "trexhealthadder"
+	}
+	delete:
+	{
+		"OnStartTouch" "trexhitbased1Add3201-1"
+	}
+	insert:
+	{
+		"OnStartTouch" "trexhitbased1,Add,160,1,-1"
+	}
+}
```

Comment:
Not currently interested in taking this change, as I think the boss health is balanced fine. We don't automatically take these from Mapea's repo, and we have even reverted them in the past when taken without review (see bathroom sponge boss).